doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -19,10 +22,13 @@ Desktop 2023 R2.
 
 Error installing PyAEDT using pip
 ---------------------------------
-1. **Proxy Server** If your company uses a proxy server you may have to update some settings at the command line.
+1. **Proxy Server** If your company uses a proxy server you may have to update proxy
+   settings at the command line.
    See the `pip documentation <https://pip.pypa.io/en/stable/user_guide/#using-a-proxy-server>`_ for details.
 2. **Install Permission** Make sure you have write access to the directory where the Python interpreter is
-   installed. This is more reason to use a `virtual environment <https://docs.python.org/3/library/venv.html>`_.
+   installed. The use of a `virtual environment <https://docs.python.org/3/library/venv.html>`_ helps
+   mitigate this issue by placing the Python interpreter and dependencies in a location that is owned
+   by the user.
 3. **Firewall** Some corporate firewalls may block pip. If you face this issue you'll have to work with your IT
    administrator to enable pip. The proxy server settings (described above) allow you to explicitly define
    the ports used by pip.
```

Comment:
```suggestion
- **Firewall**: Some corporate firewalls may block pip. If you face this issue, you'll have to work with your IT
   administrator to enable pip. The proxy server settings (described earlier) allow you to explicitly define
   the ports used by pip.
```
You don't want to use locaters like above or below because they might not be accurate in a PDF (which is generated as an artifact).